packages/dependency-operator-filter/contracts/OperatorFilterSubscription.sol

```diff
@@ -6,7 +6,7 @@ import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
 
 /// @title OperatorFilterSubription
 /// @author The Sandbox
-/// @notice This contract is ment to register and copy the default subscription of the openSea for the operator filter and our Token contract are supposed to subscribe to This contract on openSea operator filter registry
+/// @notice This contract is meant to register and copy the default subscription of the openSea for the operator filter and our Token contract are supposed to subscribe to This contract on openSea operator filter registry
```

Comment:
This whole sentence is a bit confusing at best 🤔

`openSea` => `OpenSea`
`subscribe to This contract` => `subscribe to this contract`